openquakeplatform/test/ghec_test.py

```diff
@@ -0,0 +1,63 @@
+#!/usr/bin/env python
+import unittest
+import os
+from openquake.moon import platform_get
+
+
+class GhecTest(unittest.TestCase):
+    def ghec_test(self):
+
+        pla = platform_get()
+
+        pla.get('/explore')
+        pla.wait_new_page("//b[contains(text(),"
+                          " 'Seismic Hazard Data Sets and Models')]",
+                          "/explore", strategy="next", timeout=10)
+
+        # <li>
+        # <a href="/maps/23">
+        enter_button = pla.xpath_finduniq(
+            "//li/a[normalize-space(text()) = 'Global "
+            "Historical Earthquake Catalogue "
+            "and Archive (1000-1903)']")
+
+        href = enter_button.get_attribute('href')
+        id_map = os.path.basename(href)
+        enter_button.click()
+      
+
+        pla.wait_new_page(enter_button, '/maps/%s' % id_map)
+
+        enter_button = pla.xpath_finduniq(
+            "//a[@href='/maps/%s/view' and "
+            "normalize-space(text()) = 'View Map']" % id_map)
+        enter_button.click()
+        pla.wait_new_page(enter_button, '/maps/%s/view' % id_map, timeout=15)
+
+        import time
+        time.sleep(5)
```

Comment:
sleep required ?

Reply:
Fixed this here: https://github.com/gem/oq-platform2/pull/46/commits/1d69b5794311d8dbf5821663bc452ec4a342e714